ansible/roles/builder/tasks/main.yml

```diff
@@ -170,6 +170,28 @@
     version: 1.8.3
     state: present
 
+- name: Install go-msi
+  win_chocolatey:
+    name: go-msi
+    version: '1.0.2'
+    state: present
+
+- name: Install dep
+  win_get_url:
+    url: 'https://github.com/golang/dep/releases/download/v0.3.2/dep-windows-amd64'
```

Comment:
I think that, if we download it like this, then `dep` will be available in `PATH` as `dep-windows-amd64` and not as `dep`.

Reply:
Good point